.github/workflows/format-native.yml

```diff
@@ -24,3 +24,20 @@ jobs:
     - name: Format native code
       shell: bash
       run: ./scripts/format-native.sh
+
+  check-native-headers:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3.3.0
+      - uses: actions/setup-go@v3.5.0
+        with:
+          go-version: '1.20.1'
```

Comment:
Do you prefer to use a container image or use Go to build the binary? I think that having a script that uses a container image would be easier for local usage.